libs/sdk-ui-filters/styles/scss/attributeFilterNext/attributeFilterDropdownButton.scss

```diff
@@ -19,6 +19,7 @@
     cursor: pointer;
     user-select: none;
     border-radius: 3px;
+    margin: 7px 11px 7px 0;
```

Comment:
This is quite suspicious, will it break the button when used outside of dashboard?

Reply:
It needs to be tested in AD looks like it could brake it ...